game.py

```diff
@@ -18,8 +18,8 @@ def play_game(self):
         self.players.append(Player('*'))
         print('Player 2, what is your name?')
         self.players.append(Player('/'))
-        while True:
-            try:
+        while True:  
+            try:    # while loop
```

Comment:
The try/except block should be around just the bits of code that you know should be raising exceptions.   In this case, around the add_column method.  Put this in a while loop so that if there is an error it just keeps asking for input.

If there is an error in some other method, you WANT the program to crash (so you can find the bug)